website/routes.py

```diff
@@ -1093,6 +1093,15 @@ def make_url_map(app):
         Rule('/institutions/<inst_id>/', 'get', institution_views.view_institution, OsfWebRenderer('institution.mako', trust=False))
     ])
 
+    process_rules(app, [
+        Rule([
+            '/institutions/<inst_id>/dashboard/',
+        ],
+            'get',
+            institution_views.view_institution_dashboard,
```

Comment:
If we wanted to get fancy, we could probably just pass in `use_ember_app` here rather than make an intermediate view.

Reply:
I was following convention. I'm fine with both.

Reply:
I'm also fine with both, it's just something that occurred to me.